modules/passkey/test/4337/experimental/SafeWebAuthnSharedSigner.spec.ts

```diff
@@ -94,11 +103,47 @@ describe('SafeWebAuthnSharedSigner', () => {
         expect(await sharedSigner.getConfiguration(mockAccount)).to.deep.equal([0n, 0n, 0n])
       }
     })
+
+    it('Should store the signer configuration in a mapping with the shared signer address as a key', async () => {
+      const { safeSingleton, fallbackHandler, proxyFactory, sharedSigner, mockVerifier, sharedSignerAccessor } = await setupTests()
+
+      const config = {
+        x: ethers.id('publicKey.x'),
+        y: ethers.id('publicKey.y'),
+        verifiers: ethers.toBeHex(await mockVerifier.getAddress(), 32),
+      }
+
+      const initializer = safeSingleton.interface.encodeFunctionData('setup', [
+        [sharedSigner.target],
+        1,
+        sharedSigner.target,
+        sharedSigner.interface.encodeFunctionData('configure', [config]),
```

Comment:
Doubt: For a 2/2 owner setup, won't it require something like a dispatcher (or some other better name) to call all shared signers when multiple instances are involved? Or am I getting it wrong?

Something like:
```
        [sharedSignerA.target, sharedSignerB.target],
        2,
        sharedSignerDispatcher.target,
        sharedSignerDispatcher.interface.encodeFunctionData('configureSharedSigners', [configSharedSigners]),
```

Reply:
Ah, that makes perfect sense. Thanks!